eessi_bot_event_handler.py

```diff
@@ -60,10 +71,151 @@ def handle_issue_comment_event(self, event_info, log_file=None):
         """
         request_body = event_info['raw_request_body']
         issue_url = request_body['issue']['url']
-        comment_author = request_body['comment']['user']['login']
-        comment_txt = request_body['comment']['body']
-        self.log("Comment posted in %s by @%s: %s", issue_url, comment_author, comment_txt)
-        self.log("issue_comment event handled!")
+        action = request_body['action']
+        sender = request_body['sender']['login']
+        owner = request_body['comment']['user']['login']
+        repo_name = request_body['repository']['full_name']
+        pr_number = request_body['issue']['number']
+
+        # FIXME add request body text (['comment']['body']) to log message when
+        # log level is set to debug
+        self.log(f"Comment in {issue_url} (owned by @{owner}) {action} by @{sender}")
+
+        # obtain app name and format for reporting about received & processed
+        # commands
+        app_name = self.cfg[GITHUB][APP_NAME]
+        command_response_fmt = self.cfg[BOT_CONTROL][COMMAND_RESPONSE_FMT]
+
+        # currently, only commands in new comments are supported
+        #  - commands have the syntax 'bot: COMMAND [ARGS*]'
+
+        # first check if sender is authorized to send any command
+        # - this serves a double purpose:
+        #   1. check permission
+        #   2. skip any comment updates that were done by the bot itself
+        #      --> thus we prevent the bot from entering an endless loop
+        #          where it reacts on updates to comments it made itself
+        #      NOTE this assumes that the sender of the event is corresponding to
+        #      the bot if the bot updates comments itself and that the bot is not
+        #      given permission in the configuration setting 'command_permission'
+        #      ... in order to prevent surprises we should be careful what the bot
+        #      adds to comments, for example, before updating a comment it could
+        #      run the update through the function checking for a bot command.
+        if check_command_permission(sender) is False:
```

Comment:
Since this is triggered for *every* issue comment being made, it's done too early, since for accounts that don't have permission to send commands to the bot this will immediately trigger a `account XXX has NO permission to send commands to the bot` (see https://github.com/boegel/software-layer/pull/24#issuecomment-1576845077).

That should only be done if the comment effectively includes bot commands.

Let's try and make this the first thing we fix after merging this PR...